src/test/java/com/org/jenkins/Custom/Jenkins/Distribution/Service/generators/PackageConfigGeneratorTest.java

```diff
@@ -0,0 +1,19 @@
+package com.org.jenkins.Custom.Jenkins.Distribution.Service.generators;
+
+
+import com.org.jenkins.Custom.Jenkins.Distribution.Service.Util.Util;
+import java.nio.file.Files;
+import org.json.JSONObject;
+import org.junit.jupiter.api.Test;
+
+import static com.org.jenkins.Custom.Jenkins.Distribution.Service.generators.PackageConfigGenerator.generatePackageConfig;
+
+public class PackageConfigGeneratorTest {
+    Util util = new Util();
+
+    @Test
+    public void testPackageConfigGeneration() throws Exception {
```

Comment:
You probably want to try/catch and `assertFail` when you catch an exception during testing.  Definitely add a check for the expected package config.

Reply:
Yeah just working on that.